hibernate-core/src/main/java/org/hibernate/generator/internal/CurrentTimestampGeneration.java

```diff
@@ -68,24 +71,27 @@
  * @author Gavin King
  */
 public class CurrentTimestampGeneration implements BeforeExecutionGenerator, OnExecutionGenerator {
+
+	public static final String CLOCK_SETTING_NAME = "hibernate.clock";
```

Comment:
Maybe we can make the setting string be more ORM-specific? (I know that Validator also has a use of clock and clock provider)

Reply:
This setting is provided to Hibernate ORM via `setProperty` since the value must be a `java.time.Clock` object. I don't see a need to prefix config names with `orm`, at the end of the day, all the other configs we have also aren't prefixed.